content/learn/development-guides/analysis-applications.md

```diff
@@ -1,79 +1,73 @@
 ---
 path: "/learn/development-guides/analysis-applications"
 date: "2018-05-03"
-title: "Analysis Applications"
-subTitle: "Technical resources to support the development and integration of your analysis applications."
+title: "Introduction"
+subTitle: "Technical resources to support the development and integration of analysis applications."
 ---
 
-# Creating Solutions using the DCP
+# Supporting development and integration of applications
 
-The Human Cell Atlas Data Coordination Platform holds much potential to support and enrich solutions for single cell genomics. This page highlights resources you can leverage to enable your creativity.
+The Human Cell Atlas Data Coordination Platform aims to support and encourage cellular resolution research in a variety of biological and computational areas. This page highlights available resources for enabling these areas of scientific exploration.
 
```

Comment:
This page highlights available resources to enable development of tools and methodology.

Reply:
Thanks for the suggestion! Changed.

If this page is supposed to be focused on tool development, then the Working with Ingestion Service section probably doesn't belong here. Thoughts?

Reply:
I think it's there for people wanting to build tools or packages that function with the ingest process or that require an understanding of ingest.. but I don't have a good handle on that and can ask Tim.

Reply:
Ok, we can keep it in for now.